.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## What type of issue would you like to report?
+
+- [ ] Bug
+- [ ] Feature request
+- [ ] Other, please explain:
+
+## Check if your issue is already reported
```

Comment:
Made it a link (not sure if making header a link is a good practice though), I will update the docs in a proper repo and link the PR.